src/nft/components/collection/Card.tsx

```diff
@@ -199,11 +236,11 @@ const Image = ({ uniformHeight, setUniformHeight }: ImageProps) => {
   const isMobile = useIsMobile()
 
   if (noContent) {
-    return <NoContentContainer uniformHeight={uniformHeight} />
+    return <NoContentContainer uniformHeight={uniformHeight ?? UniformHeights.unset} />
   }
 
   return (
-    <Box display="flex" overflow="hidden" borderTopLeftRadius="20" borderTopRightRadius="20">
+    <Box display="flex" overflow="hidden" borderTopLeftRadius={BORDER_RADIUS} borderTopRightRadius={BORDER_RADIUS}>
```

Comment:
Still seeing the gap between cards and the border as per https://uniswaplabs.atlassian.net/browse/WEB-2089

Reply:
ooh thanks... just made some adjustments to the radius's in the styling in card.css.ts. It looks fixed to me now visually.